languages/js/src/Oso.ts

```diff
@@ -244,22 +263,25 @@ export class Oso<
    * @returns A query that selects authorized resources of type `resourceCls`
    */
   async authorizedQuery(
-    actor: any,
-    action: any,
-    resourceCls: any
-  ): Promise<any> {
+    actor: Actor,
+    action: Action,
+    resourceCls: Class
+  ): Promise<unknown> {
```

Comment:
Would be cool to link this up with the `Query` type param if we end up moving data filtering query params into the Oso constructor.

Reply:
That would be cool but if we want to support different data sources then I think the query type would have to come through the resource.